pkg/autodiscovery/listeners/common.go

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2017-present Datadog, Inc.
 
+// +build !serverless
```

Comment:
Not familiar with the serverless build but why only this file in the whole autodiscovery package?